--- ekiga-2.0.9/configure.orig	2007-04-05 11:18:14.000000000 +0200
+++ ekiga-2.0.9/configure	2007-04-05 12:37:25.000000000 +0200
@@ -22073,7 +22073,7 @@
   test "$cross_compiling" = yes &&
   { { echo "$as_me:$LINENO: error: cannot check for file existence when cross compiling" >&5
 echo "$as_me: error: cannot check for file existence when cross compiling" >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); }; }
 if test -r "${with_pwlib_dir}/include/ptlib/pprocess.h"; then
   eval "$as_ac_File=yes"
 else
@@ -22262,7 +22262,7 @@
   test "$cross_compiling" = yes &&
   { { echo "$as_me:$LINENO: error: cannot check for file existence when cross compiling" >&5
 echo "$as_me: error: cannot check for file existence when cross compiling" >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); }; }
 if test -r "${with_pwlib_dir}/bin/ptlib-config"; then
   eval "$as_ac_File=yes"
 else
@@ -22276,7 +22276,7 @@
   HAS_PTLIB_CONFIG=1
 fi
 
-if test "x${HAS_PTLIB_CONFIG}" != "x1" ; then
+if test "x${HAS_PTLIB_CONFIG}" != "x1" && test "x${win32}" != "x1"; then
         { { echo "$as_me:$LINENO: error: Sorry but ptlib-config can not be found, please check your installation" >&5
 echo "$as_me: error: Sorry but ptlib-config can not be found, please check your installation" >&2;}
    { (exit 1); exit 1; }; };
@@ -22322,7 +22322,10 @@
 
 
 fi
-ARCH_OPAL_CFLAGS="$ARCH_OPAL_CFLAGS `$PTLIB_CONFIG --ccflags`"
+
+if test "x${win32}" != "x1"; then
+  ARCH_OPAL_CFLAGS="$ARCH_OPAL_CFLAGS `$PTLIB_CONFIG --ccflags`"
+fi
 
 
 
@@ -22350,7 +22353,7 @@
   test "$cross_compiling" = yes &&
   { { echo "$as_me:$LINENO: error: cannot check for file existence when cross compiling" >&5
 echo "$as_me: error: cannot check for file existence when cross compiling" >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); }; }
 if test -r "${with_opal_dir}/include/opal/opal/call.h"; then
   eval "$as_ac_File=yes"
 else
@@ -22383,7 +22386,7 @@
   test "$cross_compiling" = yes &&
   { { echo "$as_me:$LINENO: error: cannot check for file existence when cross compiling" >&5
 echo "$as_me: error: cannot check for file existence when cross compiling" >&2;}
-   { (exit 1); exit 1; }; }
+   { (exit 1); }; }
 if test -r "${with_opal_dir}/include/opal/call.h"; then
   eval "$as_ac_File=yes"
 else
