diff -ur src/opal/plugins/video/common/dyna.cxx opal/plugins/video/common/dyna.cxx
--- src/opal/plugins/video/common/dyna.cxx	2009-06-09 17:47:35.000000000 +0200
+++ opal/plugins/video/common/dyna.cxx	2009-06-21 16:54:13.000000000 +0200
@@ -223,7 +223,7 @@
     return false;
   }
 
-  if (seperateLibAvutil && !libAvutil.Open("avutil-49")) {
+  if (seperateLibAvutil && !(libAvutil.Open("avutil-50") || libAvutil.Open("avutil-49")) ) {
     TRACE (1, _codecString << "\tDYNA\tFailed to load FFMPEG libavutil library");
     return false;
   }
