diff -ur opal.pre/plugins/video/H.263-1998/h263-1998.cxx opal/plugins/video/H.263-1998/h263-1998.cxx
--- opal.pre/plugins/video/H.263-1998/h263-1998.cxx	2009-06-09 22:08:50.361264887 +0200
+++ opal/plugins/video/H.263-1998/h263-1998.cxx	2009-06-09 22:26:29.357035570 +0200
@@ -46,6 +46,10 @@
 
  */
 
+#ifndef PLUGIN_CODEC_DLL_EXPORTS
+#include "plugin-config.h"
+#endif
+
 #define _CRT_SECURE_NO_DEPRECATE
 
 #include "h263-1998.h"
diff -ur opal.pre/plugins/video/H.263-1998/Makefile.in opal/plugins/video/H.263-1998/Makefile.in
--- opal.pre/plugins/video/H.263-1998/Makefile.in	2009-06-09 22:08:50.361264887 +0200
+++ opal/plugins/video/H.263-1998/Makefile.in	2009-06-09 22:24:50.941032598 +0200
@@ -33,6 +33,7 @@
 COMMONDIR	=../common
 PLUGINDIR	=../../
 OBJDIR 		=./obj
+PLUGINDIR	=../..
 
 CC	 	=@CC@
 CFLAGS	 	=@CFLAGS@ 
@@ -42,6 +43,7 @@
 STDCCFLAGS	=@STDCCFLAGS@
 LIBAVCODEC_CFLAGS=@LIBAVCODEC_CFLAGS@
 LDFLAGS		=@LDFLAGS@ -lavcodec -lavutil
+EXTRACFLAGS	=-I$(PLUGINDIR)
 HAVE_LIBAVCODEC_RTP_MODE=@HAVE_LIBAVCODEC_RTP_MODE@
 
 ifdef HAVE_LIBAVCODEC_RTP_MODE
@@ -65,7 +67,7 @@
 
 $(OBJDIR)/%.o : %.cxx
 	@mkdir -p $(OBJDIR) >/dev/null 2>&1
-	$(CXX) -I../../../include $(EXTRACCFLAGS) $(STDCCFLAGS) $(OPTCCFLAGS) $(CFLAGS) -c $< -o $@
+	$(CXX) -I../../../include $(EXTRACCFLAGS) $(STDCCFLAGS) $(OPTCCFLAGS) $(CFLAGS) $(EXTRACFLAGS) -c $< -o $@
 
 PLUGIN	= ./$(BASENAME)_video_pwplugin.$(PLUGINEXT)
 STATIC	= ./lib$(BASENAME)_video_s.a
