diff -r --exclude .git -u ../sshtrunk/lib/engine/gui/gtk-frontend/chat-area.cpp ekiga/lib/engine/gui/gtk-frontend/chat-area.cpp
--- ../sshtrunk/lib/engine/gui/gtk-frontend/chat-area.cpp	2014-05-11 18:40:22.058387345 +0200
+++ ekiga/lib/engine/gui/gtk-frontend/chat-area.cpp	2014-05-15 11:26:41.145462923 +0200
@@ -831,8 +831,8 @@
 		      FALSE, TRUE, 2);
 
   /* the BOLD button */
-  button = gtk_button_new_from_icon_name ("format-text-bold",
-					  GTK_ICON_SIZE_BUTTON);
+  button = gtk_button_new ();//_from_icon_name ("format-text-bold",
+  //GTK_ICON_SIZE_BUTTON);
   g_object_set_data_full (G_OBJECT (button), "gm_open_tag",
 			  (gpointer) "<b>", NULL);
   g_object_set_data_full (G_OBJECT (button), "gm_close_tag",
@@ -846,8 +846,8 @@
   gtk_widget_show (button);
 
   /* the ITALIC button */
-  button = gtk_button_new_from_icon_name ("format-text-italic",
-					  GTK_ICON_SIZE_BUTTON);
+  button = gtk_button_new ();//"format-text-italic",
+  //GTK_ICON_SIZE_BUTTON);
   g_object_set_data_full (G_OBJECT (button), "gm_open_tag",
 			  (gpointer) "<i>", NULL);
   g_object_set_data_full (G_OBJECT (button), "gm_close_tag",
@@ -861,8 +861,8 @@
   gtk_widget_show (button);
 
   /* the UNDERLINE button */
-  button = gtk_button_new_from_icon_name ("format-text-underline",
-					  GTK_ICON_SIZE_BUTTON);
+  button = gtk_button_new ();//_from_icon_name ("format-text-underline",
+  //GTK_ICON_SIZE_BUTTON);
   g_object_set_data_full (G_OBJECT (button), "gm_open_tag",
 			  (gpointer) "<u>", NULL);
   g_object_set_data_full (G_OBJECT (button), "gm_close_tag",
diff -r --exclude .git -u ../sshtrunk/lib/engine/gui/gtk-frontend/chat-window.cpp ekiga/lib/engine/gui/gtk-frontend/chat-window.cpp
--- ../sshtrunk/lib/engine/gui/gtk-frontend/chat-window.cpp	2014-03-03 16:04:22.393857464 +0100
+++ ekiga/lib/engine/gui/gtk-frontend/chat-window.cpp	2014-05-15 11:26:41.145462923 +0200
@@ -140,7 +140,7 @@
 {
   self->priv->visible_conversation_connection.disconnect ();
 
-  GtkWidget* page = gtk_stack_get_visible_child (GTK_STACK (self->priv->stack));
+  /*GtkWidget* page = gtk_stack_get_visible_child (GTK_STACK (self->priv->stack));
 
   if (IS_CONVERSATION_PAGE (page)) {
 
@@ -148,7 +148,7 @@
 
     self->priv->visible_conversation_connection = conversation->updated.connect (boost::bind (&on_visible_conversation_updated, self));
     chat_window_update_menu (self, conversation);
-  }
+    }*/
 }
 
 static void
@@ -195,9 +195,9 @@
   g_signal_connect (page, "updated",
 		    G_CALLBACK (on_unread_count_updated), self);
 
-  gtk_stack_add_titled (GTK_STACK (self->priv->stack),
+  /*gtk_stack_add_titled (GTK_STACK (self->priv->stack),
 			page, name.c_str (),
-			conversation_page_get_title (CONVERSATION_PAGE (page)));
+			conversation_page_get_title (CONVERSATION_PAGE (page)));*/
   gtk_widget_show_all (page);
 
   self->priv->connections.add (conversation->user_requested.connect (boost::bind (&on_some_conversation_user_requested, self, name)));
@@ -209,7 +209,7 @@
 on_some_conversation_user_requested (ChatWindow* self,
 				     const std::string name)
 {
-  gtk_stack_set_visible_child_name (GTK_STACK (self->priv->stack), name.c_str ());
+  //gtk_stack_set_visible_child_name (GTK_STACK (self->priv->stack), name.c_str ());
   gtk_widget_show (GTK_WIDGET (self));
   gtk_window_present (GTK_WINDOW (self));
 }
@@ -320,10 +320,10 @@
 		      FALSE, FALSE, 0);
   gtk_widget_show (self->priv->gear);
 
-  GtkWidget* switcher = gtk_stack_switcher_new ();
+  /*GtkWidget* switcher = gtk_stack_switcher_new ();
   gtk_box_pack_start (GTK_BOX (hbox), switcher,
 		      FALSE, TRUE, 0);
-  gtk_widget_show (switcher);
+              gtk_widget_show (switcher);
 
   self->priv->stack = gtk_stack_new ();
   gtk_stack_switcher_set_stack (GTK_STACK_SWITCHER (switcher), GTK_STACK (self->priv->stack));
@@ -332,7 +332,7 @@
   gtk_widget_show (self->priv->stack);
   g_signal_connect (self->priv->stack, "notify::visible-child",
 		    G_CALLBACK (on_visible_conversation_changed), self);
-
+  */
   boost::shared_ptr<Ekiga::ChatCore> chat_core =
     core.get<Ekiga::ChatCore> ("chat-core");
   self->priv->connections.add (chat_core->dialect_added.connect (boost::bind (&on_dialect_added, self, _1)));
diff -r --exclude .git -u ../sshtrunk/lib/engine/gui/gtk-frontend/conversation-page.cpp ekiga/lib/engine/gui/gtk-frontend/conversation-page.cpp
--- ../sshtrunk/lib/engine/gui/gtk-frontend/conversation-page.cpp	2014-03-03 16:04:22.393857464 +0100
+++ ekiga/lib/engine/gui/gtk-frontend/conversation-page.cpp	2014-05-15 11:26:41.145462923 +0200
@@ -90,9 +90,9 @@
     self->priv->title = g_strdup_printf ("%s",
 					 self->priv->conversation->get_title ().c_str ());
   }
-  gtk_header_bar_set_title (GTK_HEADER_BAR (self->priv->header), self->priv->title);
-  gtk_header_bar_set_subtitle (GTK_HEADER_BAR (self->priv->header),
-			       self->priv->conversation->get_status ().c_str ());
+  //gtk_header_bar_set_title (GTK_HEADER_BAR (self->priv->header), self->priv->title);
+  //gtk_header_bar_set_subtitle (GTK_HEADER_BAR (self->priv->header),
+  //		       self->priv->conversation->get_status ().c_str ());
 }
 
 static void
@@ -151,11 +151,11 @@
 
   result->priv->connections.add (conversation->updated.connect (boost::bind (&on_conversation_updated, result)));
 
-  header = gtk_header_bar_new ();
-  result->priv->header = header;
-  gtk_box_pack_start (GTK_BOX (result), header,
-		      FALSE, FALSE, 2);
-  gtk_widget_show (header);
+  //header = gtk_header_bar_new ();
+  //result->priv->header = header;
+  //gtk_box_pack_start (GTK_BOX (result), header,
+  //	      FALSE, FALSE, 2);
+  //gtk_widget_show (header);
 
   box = gtk_box_new (GTK_ORIENTATION_HORIZONTAL, 2);
   gtk_box_pack_start (GTK_BOX (result), box,
